error: field ‘pkt6’ has incomplete type (with nginx 1.23.2)

building nchan on Archlinux against nginx-mainline 1.23.2 fails with:
```
        -o objs/addon/util/nchan_accumulator.o \
        ../nchan-1.3.5/src/util/nchan_accumulator.c
In file included from src/event/ngx_event.h:526,
                 from src/http/ngx_http_upstream.h:14,
                 from src/http/ngx_http.h:34,
                 from ../nchan-1.3.5/src/nchan_module.h:17,
                 from ../nchan-1.3.5/src/util/nchan_accumulator.c:2:
src/event/ngx_event_udp.h:38:27: error: field ‘pkt6’ has incomplete type
   38 |     struct in6_pktinfo    pkt6;
      |                           ^~~~
make[1]: *** [objs/Makefile:1499: objs/addon/util/nchan_accumulator.o] Error 1
make[1]: Leaving directory '/home/damjan/projects/aur/nginx-mainline-mod-nchan/src/build'
make: *** [Makefile:16: modules] Error 2
```

Researching the issue, I've come to this:
https://trac.nginx.org/nginx/ticket/2312

which refers to http://nginx.org/en/docs/dev/development_guide.html#include_files
>  The following two #include statements must appear at the **beginning** of every nginx file: 

(emphasis on "beginning" mine)

ps. I can probably create a PR later in the week.

--- a/src/nchan_module.h
+++ b/src/nchan_module.h
@@ -13,8 +13,10 @@
 //#define ONE_FAKE_CHANNEL_OWNER 2
 #define MAX_FAKE_WORKERS 5
 #endif
-#include <nchan_version.h>
+#include <ngx_config.h>
+#include <ngx_core.h>
 #include <ngx_http.h>
+#include <nchan_version.h>
 
 //building for old versions
 #ifndef NGX_MAX_INT_T_VALUE

